contracts/Ethleverage.sol

```diff
@@ -71,24 +79,31 @@ contract Ethleverage {
     pethContract = _pethaddr;
     MKRContract = _mkrContract;
     makerLR = _liquidationRatio;
-
-
-
-
   }
 
 	function initialize() public {
-
-			ICDPContract(CDPContract).approve(address(this));
-
+      //for some reason the variables ARE initialized in the constructor function, but an error is thrown without reassigning them here.
+      owner = msg.sender;
+      moneyMakerKovan = 0xd87856163f409777df41DDFBF37A66369E028FA9;
+      liquidKovan =  0xc936749D2D0139174EE0271bD28325074fdBC654; //https://github.com/makerdao/sai/blob/4b0c94b8ef2d8e0951dd0a0eee7c0fce5f5dbb49/src/tap.sol
```

Comment:
What's this liquidKovan contract?

Reply:
It is to purchase PETH available from forced CDP liquidations. I was using it as an alternative to the illiquid OasisDex market that runs on Kovan... However, there is no more PETH available for sale anymore on the Kovan network.  
![image](https://user-images.githubusercontent.com/6872751/36701725-67163846-1b22-11e8-99b9-6b69d0b34a37.png)